b3lb/rest/models.py

```diff
@@ -408,9 +408,51 @@ def records_effective_hold_time(self):
 class SecretAdmin(ModelAdmin):
     model = Secret
     list_filter = [('tenant', RelatedOnlyFieldListFilter)]
-    list_display = ['__str__', 'description', 'endpoint', 'recording_enabled', 'attendee_limit', 'meeting_limit']
+    list_display = ['__str__', 'description', 'endpoint', 'api_mate', 'recording_enabled', 'attendee_limit', 'meeting_limit', ]
     actions = [records_on, records_off]
 
+    def api_mate(self, obj):
+        low_slug = str(obj.tenant.slug).lower()
+        low_slug_id = f"{low_slug}-{str(obj.sub_id).zfill(3)}"
+        params = {
+            "sharedSecret": obj.secret,
+            "name": f"API Mate test room for {low_slug_id}",
+            "attendeePW": get_random_string(6, '0123456789'),
+            "moderatorPW": get_random_string(6, '0123456789')
+        }
+        slide_string = ""
+        try:
+            slide = Asset.objects.get(tenant__secret=obj).slide
+            if slide:
+                slide_string = f"pre-upload=https://{st.B3LB_API_BASE_DOMAIN}/b3lb/t/{low_slug}/slide"
+        except Asset.DoesNotExist:
+            pass
+
+        if obj.recording_enabled and obj.tenant.recording_enabled:
+            params["record"] = True
```

Comment:
I wonder if:
- we assume that recording is enabled by default and only set `record` to false if appropriate
- always set `record` to the result of the *if* condition

It should be identical to the `create` api callout.

Reply:
If it is set to "False" it will be marked as checks in the webpage. It seems it only evaluates if the parameter is given or not. If it's enabled, we can set it otherwise don't add it to the link parameters.

Additionally the if statement can be checked via property statement from Secret model.